docs/iris/src/whatsnew/2.2.rst

```diff
@@ -66,6 +66,9 @@ Iris 2.2 Features
   discontiguous points in coordinates can be explicitly masked
   using another new feature :func:`iris.util.mask_cube`.
 
+* :func:`iris.util.array_equal` now has a 'withnans' keyword, which provides
+  a NaN-tolerant array comparison.
+
```

Comment:
I'm torn with this.
Part of me wants it in under a sub-heading, similar to the ["bugs fixed in 1.7.3" sub headings](https://scitools.org.uk/iris/docs/latest/whatsnew/1.7.html#bugs-fixed-in-v1-7-3) in the 1.7 what's new but I'm not sure...

Reply:
Hmm. It's awkward because bugfix releases aren't _supposed_ to contain new features !

We could treat the new keyword as a private matter, and announce the new functionality with enormous fanfare in the nest minor release ??
I think that means putting the whatsnew contribution back into a textfile in a contributions_2.3.0 directory.
The actual code changes (+docstring) can stay though,

Would you favour that solution @lbdreyer  ?

Reply:
I believe the outcome of the offline discussion was to keep what you have already done.
Admittedly the reasons for that decision are a little fuzzy we discussed this over a week ago, but I think I personally wanted to not keep it private, but not add confusion by adding a "features added in 2.2.1" section

So basically, I'm happy with this change as it is.